server/models/studentModel.js

```diff
@@ -0,0 +1,39 @@
+
+const mongoose = require('mongoose')
+const Schema = mongoose.Schema;
+
+const studentSchema = new Schema({
+
+    fName: {
```

Comment:
is there a reason to call it fName instead of name?

Reply:
No relevant big reason, It was name before then it was conflicted by mistake with something else while defining them  in the controller so i just changed it to firstname...